packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -96,5 +96,10 @@
   },
   "engines": {
     "node": ">= 8.10.0"
+  },
+  "awslint": {
+    "exclude": [
+      "grant-result:@aws-cdk/aws-lambda.LayerVersionBase.grantUsage"
```

Comment:
Because also there the code there doesn't really use IAM policies, doesn't even use principals, and I didn't really want to spend time on this right now:

https://github.com/awslabs/aws-cdk/blob/master/packages/@aws-cdk/aws-lambda/lib/layers.ts#L79